exporter/awsxrayexporter/translator/segment_test.go

```diff
@@ -30,6 +30,8 @@ var (
 	testWriters = newWriterPool(2048)
 )
 
+// TODO: add unit test to test 30 days expiration functionality (Cong)
```

Comment:
Keep this in your mind but remove the TODO we use TODO to mark when, for example, taking on some technical debt and possibly revisit (in practice rarely ;) ) later. But it's not for task list